front/public/locales/en/operationalStudies/project.json

```
@@ -7,7 +7,15 @@
   "error": {
     "unableToDeleteProject": "Unable to delete project",
     "unableToPostDocumentTitle": "Failure to upload the picture",
-    "unableToPostDocument": "Unable to load the picture"
+    "unableToPostDocument": "Unable to load the picture",
+    "unableToUpdateProjectTitle": "Failure to update the project",
+    "unableToUpdateProject": "Unable to update project",
+    "unableToCreateProjectTitle": "Failure to update the project",
+    "unableToCreateProject": "Unable to update project",
+    "uploadImageSizeTitle": "Image size is too large",
+    "uploadImageSize": "The image size exceeds the maximum size of 2MB",
```

To avoid repeating "size"

```suggestion
    "uploadImageSize": "The maximum image size is 2MB",
```